src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -27,7 +27,8 @@ class CrossDeviceLink extends Component {
     super(props)
 
     if (!props.socket) {
-      const socket = createSocket()
+      const url = props.urls.sync_url
```

Comment:
small inconsistency: in other similar cases deconstruction is used. To be honest, this approach look cleaner, it's just that it's the odd one out.

Reply:
You are right, this is definitely more readable. Will change it the assignment for the other cases